requirements.txt

```diff
@@ -1,82 +1,84 @@
-aiohttp==3.4.4
+aiohttp==3.5.4
 appnope==0.1.0
 argh==0.26.2
 async-timeout==3.0.1
-attrs==18.2.0
+attrs==19.1.0
 Babel==2.6.0
 backcall==0.1.0
-bleach==3.0.2
+bleach==3.1.0
 blinker==1.4
-certifi==2018.10.15
+certifi==2019.3.9
 chardet==3.0.4
-cloudpickle==0.6.1
-decorator==4.3.0
+cloudpickle==0.8.0
+decorator==4.3.2
 defusedxml==0.5.0
 docutils==0.14
 doit==0.31.1
-entrypoints==0.2.3
+entrypoints==0.3
 ghp-import2==1.0.1
 html5lib==1.0.1
 husl==4.0.3
 icalendar==4.0.3
-idna==2.7
+idna==2.8
 idna-ssl==1.1.0
 ipykernel==5.1.0
-ipython==7.1.1
+ipython==7.3.0
 ipython-genutils==0.2.0
-jedi==0.13.1
+jedi==0.13.3
 Jinja2==2.10
-jsonschema==2.6.0
-jupyter-client==5.2.3
+jsonschema==3.0.1
+jupyter-client==5.2.4
 jupyter-core==4.4.0
-Logbook==1.4.1
-lxml==4.2.5
+Logbook==1.4.3
+lxml==4.3.2
 Mako==1.0.7
-Markdown==2.6.11
-MarkupSafe==1.1.0
-micawber==0.3.8
+Markdown==3.0.1
+MarkupSafe==1.1.1
+micawber==0.4.0
 mistune==0.8.4
-multidict==4.5.1
-natsort==5.5.0
-nbconvert==5.4.0
+multidict==4.5.2
+natsort==6.0.0
+nbconvert==5.4.1
 nbformat==4.4.0
-Nikola==8.0.1
-notebook==5.7.2
+Nikola==8.0.2
+notebook==5.7.6
 pandocfilters==1.4.2
-parso==0.3.1
+parso==0.3.4
 pathtools==0.1.2
 pexpect==4.6.0
 phpserialize==1.3
 pickleshare==0.7.5
 piexif==1.1.2
-Pillow==5.3.0
-prometheus-client==0.4.2
-prompt-toolkit==2.0.7
+Pillow==5.4.1
+prometheus-client==0.6.0
+prompt-toolkit==2.0.9
 ptyprocess==0.6.0
 pygal==2.4.0
-Pygments==2.3.0
+Pygments==2.3.1
 pyinotify==0.9.6
 Pyphen==0.9.5
+pyrsistent==0.14.11
 PyRSS2Gen==1.1
-python-dateutil==2.7.5
-pytz==2018.7
+python-dateutil==2.8.0
+pytz==2018.9
 PyYAML==3.13
-pyzmq==17.1.2
-requests==2.20.1
+pyzmq==18.0.1
+requests==2.21.0
 Send2Trash==1.5.0
 simplegeneric==0.8.1
-six==1.11.0
+six==1.12.0
 smartypants==2.0.1
 terminado==0.8.1
 testpath==0.4.2
 toml==0.10.0
-tornado==5.1.1
+tornado==6.0.1
```

Comment:
tornado 6.0.2 has a fix for
WebSocketHandler.set_nodelay
which may be helpful here